crates/holochain/src/conductor/conductor.rs

```diff
@@ -287,17 +294,32 @@ where
         port: u16,
         handle: ConductorHandle,
     ) -> ConductorResult<u16> {
-        let app_api = RealAppInterfaceApi::new(handle);
+        let interface_id: AppInterfaceId = format!("interface-{}", port).into();
+        let app_api = RealAppInterfaceApi::new(handle, interface_id.clone());
+        // This receiver is thrown away because we can produce infinite new
+        // receivers from the Sender
         let (signal_broadcaster, _r) = tokio::sync::broadcast::channel(SIGNAL_BUFFER_SIZE);
```

Comment:
SIGNAL_BUFFER_SIZE is only 5. This might be a problem if UI aren't polling draining the recv side fast enough. You will get RecvLagged error which might not be an issue if it's handled correctly but even 5 might be too small of a buffer. Might need to experiment with this.

Reply:
Bumped to 50, which might even be unnecessarily large. I agree that 5 is small, but the UI doesn't need to poll, it's the interface task in Holochain that needs to poll, so hopefully tokio is doing its job. I guess the main way it could get backed up is if we have a task that blocks for too long.